src/inputs/dropdown/dropdown.jsx

```diff
@@ -71,6 +71,37 @@ class Dropdown extends React.Component {
         if (!_.isEqual(currentPropsValue, previousPropsValue)) {
             this.setState({ menuIsOpen: false, value: currentPropsValue });
         }
+
+        const {
+            autoScrollSelection,
+            options,
+            selection,
+            selectionMatchProp,
+            value
+        } = this.props;
+        if (selection && autoScrollSelection && this.dropdownMenu && value) {
+            const itemHeight = this.dropdownMenu.getScrollHeight()/_.size(options);
+            const pageSize = this.dropdownMenu.getClientHeight()/itemHeight;
+            const selectionIndex = _.findIndex(options, (o) => {
```

Comment:
Nitpick: A blank line in between Lines 84 and 85 would be nice but this is absolutely _Not a Blocker_

Reply:
Same here.

Reply:
Thanks for fixing.  Resolving this thread.